custom/opm/beneficiary.py

```diff
@@ -564,6 +564,24 @@ def weight_grade_normal(self):
                     return self.child_age/12
                 return False
 
+    def weight_grade_status(self, status):
+        if self.status == 'pregnant':
+            return False
```

Comment:
Oh yeah, I think it's valuable to have this condition in here, by "I don't think this matters", I meant to refer to my nitpick of returning `False` instead of `None`.  The condition is a good call.